sleeper_wrapper/players.py

```diff
@@ -4,8 +4,11 @@ class Players(BaseApi):
 	def __init__(self):
 		pass
 
-	def get_all_players(self):
-		return self._call("https://api.sleeper.app/v1/players/nfl")
+	def get_all_players(self, sport):
```

Comment:
I think we should probably have `sport="nfl"` as the default argument since this would change functionality for people and I assume most users are using the package for NFL data and not NBA or LCS.

Reply:
@dtsong meant to tag you here. Any thoughts on adding support for these other sports?